### PR TITLE
Gparted 1.2.0 with deps

### PR DESCRIPTION
**Pull request by satmandu**
This is a working gparted with updated deps.

There are a bunch of packages, but gparted DOES run, which is... nice.

Works properly:
- [x] x86_64

Builds properly:
- [x] x86_64
- [x] armv7l
- [x] i686

### CODE REVIEW

File: packages/gparted.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
+require 'package'
+
+class Gparted < Package
+  description 'A Partition Magic clone, frontend to GNU Parted'
```
Comment (uberhacker) [inline packages/gparted.rb:4]: Missing homepage?

File: packages/xfsprogs.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
+require 'package'
+
+class Xfsprogs < Package
+  description 'XFS filesystem utilities'
```
Comment (uberhacker) [inline packages/xfsprogs.rb:4]: Missing homepage?

**Comment by uberhacker** [comment]
Do you ever sleep @satmandu? ;)

**Comment by satmandu** [comment]
> Do you ever sleep @satmandu? ;)

This has been percolating on and off for a couple of months.

File: packages/xfsprogs.rb (hunk 2 above)
Comment (satmandu) [reply packages/xfsprogs.rb:4]: fixed

**Comment by uberhacker** [comment]
I hope I don't regret merging this. :)

**Pull request merged**